### ISSUE TITLE
support NVMe-OF (NVMe via TCP)

### ISSUE THREAD

**Issue by TheJJ** [SFTtech/stiefelsystem#19]
possible alternative to nbd

**Comment by ojafuenf** [comment]
The server is configured via *nvmetcli*. (git://git.infradead.org/users/hch/nvmetcli.git, depends on configshell-fb)
In arch the *nvmet* kernelmodule must be loaded manually. Debian does it automatically
The configuration is documented at https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-nvmeof.html

The client only requires *nvme-cli* (from package manager).
In arch the *nvme_fabrics* and *nvme_tcp* have to be loaded manually again. Debian does it automatically.
The basic command are:
nvme discover -t tcp -a SERVERIP -s 4420 -n stiefel
nvme connect -t tcp -a SERVERIP -s 4420 -n stiefel
nvme disconnect -d /dev/nvmeX

**Comment by franciscod** [comment]
Related project that just popped and made me think of stiefelsystem: https://github.com/poettering/diskomator

(also uses NVMe-TCP)

**Comment by TheJJ** [comment]
Indeed, @poettering's idea and motivation seems very similar to ours - would be cool if we could benefit from features in other components added because of his project. It also increases our motivation to finally finish our better-packageable improved version of stiefelsystem 😊
